app/assets/stylesheets/header.scss

```diff
@@ -1,28 +1,57 @@
-.account-actions {
+.header-actions {
+  width: 20%;
+  margin-right: 4%;
   display: flex;
   flex-direction: row;
+  align-items: center;
   gap: 20px;
+  margin-left: 20px;
+  overflow: visible;
+}
+
+.h-img {
+  height: 35px;
+  width: 35px;
+}
+
+.header-actions > * {
+  display: flex;
   height: 100%;
   width: 25%;
   align-items: center;
-  margin-left: 20px;
+  overflow: visible;
+}
+
+#search-options {
+  overflow-x: visible;
+  width: 40%;
+  display: flex;
+  flex-direction: row;
 }
 
 .messages {
+  display: flex;
+  justify-content: center;
   width: 50%;
   height: 100%;
+  padding: 3px 0 3px 0;
 }
 
 .message-success {
-  background-color: green;
+  background-color: rgba(0, 255, 0, 0.5);
```

Comment:
please use consistent notation 000, 255.... usw  im not shure but i believe this is the better way